aggregator-backup/download-aggregator.sh

```diff
@@ -35,18 +34,26 @@ if [ "$r" == "${r#[y]}" ]; then
 fi
 
 # Grab the Pod Name of the aggregator pod
-# If you use zsh and this line isn't working, it is likely due to your partial line response
-# and zsh adds a % delimeter to the end. Add the following line to your ~/.zshrc file:
-# PROMPT_EOL_MARK=''
 podName="$(kubectl get pods -n "$namespace" -l app=aggregator -o jsonpath='{.items[0].metadata.name}')"
 
-# find read db file
-readDbFile="$(kubectl exec -c aggregator "$podName" -n "$namespace" -- find /var/configs/waterfowl -type f -name '*.read')"
-echo "Found read db file: $readDbFile"
+# Use ephemeral container to access and copy the read DB file
+echo "Accessing aggregator filesystem using ephemeral container..."
+kubectl debug -n "$namespace" "$podName" -it --image=busybox --target=aggregator --share-processes -- /bin/sh -c "
+  cd /proc/1/root${aggDir}
```

Comment:
Do we know for sure this is always under /proc/1/root ? 

@jessegoodier @cliffcolvin